### DISCUSSION TITLE
Use the fade animation/effect in your messagebox!

### DISCUSSION THREAD

**Discussion by Akascape** [Akascape/CTkMessagebox#8] [Show and tell]
Now you can easily set a **fade in and fade out effect** in CTkMessagebox!
Just add this option: `fade_in_duration=50`, (greater the value, faster the animation)
By default, this is turned off.

![](https://user-images.githubusercontent.com/89206401/229573390-97a0ed74-20ea-4797-9003-06a414677880.gif)

Tested properly in windows, mac, ubuntu

**Comment by avalon60** [comment]
Nice!
